networkx/readwrite/json_graph/jit.py

```diff
@@ -0,0 +1,99 @@
+"""
+Read and write NetworkX graphs as JavaScript InfoVis Toolkit (JIT) format JSON.
+
+See the JIT documentation and examples at http://thejit.org
```

Comment:
Don't use raw URLs in documentation, give a semantically appropriate link title:

    See the `JIT documentation`_ for more examples.

    .. _JIT documentation: http://thejit.org